hyperSpec/R/seq.R

```diff
@@ -76,3 +80,65 @@ wl.seq <- function(x, from = 1, to = ncol(x@data$spc), ...) {
     seq(from = from, to = to, ...)
   }
 }
+
+# Unit tests -----------------------------------------------------------------
+
+.test(seq.hyperSpec) <- function() {
+
+  context("seq.hyperSpec")
+
+  # Perform tests
+  test_that("seq.hyperSpec() works", {
+    sp <- generate_hy_spectra()
+
+    expect_equal(max(seq(sp, index = TRUE)), nrow(sp))
+    expect_equal(min(seq(sp, index = TRUE)), 1)
+
+    expect_is(seq(sp),      "hyperSpec")
+    expect_is(seq_along(sp),"integer")
+
+    expect_equal(seq_along(sp), seq(sp, index = TRUE))
+
+    x <- seq(sp, length.out = 3, index = TRUE) # return value is numeric, not integer!
```

Comment:
> ` x <- seq(sp, length.out = 3, index = TRUE) # return value is numeric, not integer!`

I took this from the examples. @cbeleites should I open a new issue for this?

Reply:
Well, since it is documented behavior (via examples) it should be fine. I cannot decide right now whether or not there may be situations where we want this. But it is easier to round if it is not wanted (and R automatically does this if the result is used for indexing) than to undo any rounding we impose automatically. => leave it as it is for now, no issue.